substrate/frame/staking-async/runtime-api/src/lib.rs

```diff
@@ -35,5 +38,8 @@ sp_api::decl_runtime_apis! {
 
 		/// Returns true if validator `account` has pages to be claimed for the given era.
 		fn pending_rewards(era: sp_staking::EraIndex, account: AccountId) -> bool;
+
+		/// Returns a list of (era, amount) that indices at which era unbonded funds will be unlocked.
+		fn unbonding_duration(account: AccountId) -> Vec<(sp_staking::EraIndex, Balance)>;
```

Comment:
Suggestion: We should expose a runtime api that that estimates `if I unbond now, when do my funds unlock?` so frontends can show the current ETA. Wdyt?